components/blocks-renderer.tsx

```diff
@@ -47,9 +45,7 @@ const componentMap = {
   LocationBlock,
   TableLayout,
   Agenda,
-  Organizer,
-  JoinGithub,
-  JoinAsPresenter,
```

Comment:
Why are these being removed?

Reply:
The page of 'User Group' may not use component <Blocks />